src/Addon/Plugin/PluginCriteria.php

```diff
@@ -21,13 +21,6 @@ class PluginCriteria
      */
     protected $model = null;
 
-    /**
-     * The cache prefix.
-     *
-     * @var null|string
-     */
-    protected $cachePrefix = null;
-
```

Comment:
I combined every cache relevant option in `$this->option['cache']` to keep it consistent. All other cache options (ttl, namespace, key) are already there.